packages/chakra-ui-core/src/CButton/utils/button.props.js

```diff
@@ -33,7 +33,7 @@ export const buttonProps = {
   size: {
     type: String,
     default: 'md',
-    validator: value => value.match(/^(sm|md|lg)$/)
+    validator: value => value.match(/^(xs|sm|md|lg)$/)
```

Comment:
Good catch, @kurotanshi ! 

To further improve this, you can also include the `xl` size to the validators. I forgot to include it therein.

https://github.com/chakra-ui/chakra-ui-vue/blob/9df688c3d15ed3e49291d83cfa26fc600e513a9c/packages/chakra-ui-core/src/CButton/utils/button.styles.js#L27-L48

Reply:
thanks for reminding!